tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -885,6 +885,18 @@ qos/test_qos_sai.py::TestQosSai::testQosSaiSharedReservationSize:
     conditions:
       - "asic_type not in ['cisco-8000']"
 
+qos/test_qos_sai.py::TestQosSai::testQosSaiLosslessVoq:
```

Comment:
already have this skip condition, just need to keep "reason" line of this PR, to describe more accurately